tests/test_large_result_set.c

```diff
@@ -3,17 +3,25 @@
  */
 #include "utils/test_setup.h"
 
-
 void test_large_result_set_helper(sf_bool use_arrow) {
 
     int rows = 100000; // total number of rows
 
     SF_STMT *sfstmt = NULL;
     SF_CONNECT *sf = setup_snowflake_connection();
-#ifdef __APPLE__
-    sf_bool insecure_mode = SF_BOOLEAN_TRUE;
-    snowflake_set_attribute(sf, SF_CON_INSECURE_MODE, &insecure_mode);
-#endif
+
+    // Sometime we can't get OCSP response from cache server or responder
```

Comment:
Do we have a task for it? Can we include the task number here?

Reply:
@sfc-gh-ext-simba-hx please add `TODO SNOW-1526335` at the beginning of the comment